### PR TITLE
Add Travis CI for flake8

### PR DESCRIPTION
**Pull request by jlausuch**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+install:
+  - pip install -e .[dev]
+script:
+  - flake8 qatrfm
+  - flake8 tests
+deploy:
```
Comment (cfconrad) [inline .travis.yml]: Are we already on pypi?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  provider: pypi
+  user: "jlausuch"
+  password: $PYPI_PASS
```
Comment (cfconrad) [inline .travis.yml]: Comes PYPI_PASS from shell environment?
What I found so far is: https://docs.travis-ci.com/user/deployment/pypi/ which has some encryption mechanism

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+dist: xenial
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+install:
+  - pip install -e .[dev]
+script:
+  - flake8 qatrfm
```
Comment (cfconrad) [inline .travis.yml]: why not `flake8 . `

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    - master
+    - /\bv[0-9]/
+dist: xenial
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+install:
+  - pip install -e .[dev]
```
Comment (cfconrad) [inline .travis.yml]: what does that `[dev]` ?

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+branches:
+  only:
+    - master
+    - /\bv[0-9]/
+dist: xenial
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7"
```
Comment (cfconrad) [inline .travis.yml]: I like, that we do not care for py2 !!

File: .travis.yml (hunk 3 above)
Comment (jlausuch) [reply .travis.yml]: ok

File: .travis.yml (hunk 1 above)
Comment (jlausuch) [reply .travis.yml]: no no, actually I will remove this section for now. We'll see in the future :)

File: .travis.yml (hunk 2 above)
Comment (jlausuch) [reply .travis.yml]: thanks for the pointer, but will remove pypi for now.

File: .travis.yml (hunk 4 above)
Comment (jlausuch) [reply .travis.yml]: something we don't need for now (it installs tox), so I'll remove it

**Pull request merged**